dcos/package.py

```diff
@@ -943,6 +946,25 @@ def copy_to_cache(self, target_dir):
                 'Unable to fetch packages from [{}]'.format(self.url))
 
 
+def _rmtree_on_error(func, path, exc_info):
```

Comment:
This *still* doesn't follow our pydoc format.   Please look at the docstrings for the other functions.